### DISCUSSION TITLE
Back gesture shows a white screen on android instead of quitting the app

### DISCUSSION THREAD

**Discussion by FredJul** [felangel/flow_builder#28] [Help]
**Describe the bug**
Since 0.0.1-dev.10, there is a regression when using the back gesture (Android 11) on the main screen.  A back should close the app, but since dev.10 it shows a white screen instead.

**To Reproduce**
1. Compile the https://github.com/FredJul/Warmd app and change flow_builder version to use the last one (currently 0.0.1-dev.11)
2. Press next on all questionnaire screen until you reached the score screen
3. Use the back gesture, it displays a white screen instead of quitting the app

**Expected behavior**
Same behavior than 0.0.1-dev.9, the back gesture on the score screen should quit the app

**Logs **
flutter doctor -v
[✓] Flutter (Channel stable, 1.22.5, on macOS 11.0.1 20B29 darwin-x64, locale fr-FR)
    • Flutter version 1.22.5 at /Users/f.julian/Documents/flutter
    • Framework revision 7891006299 (27 hours ago), 2020-12-10 11:54:40 -0800
    • Engine revision ae90085a84
    • Dart version 2.10.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/f.julian/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/f.julian/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] VS Code (version 1.52.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.17.0

**Additional context**
I believe https://github.com/felangel/flow_builder/pull/14 to be the culprit of this behavior change.

Also, https://github.com/FredJul/Warmd is not a big project (FlowBuilder is exclusively used in the main.dart file), but if you have difficulties reproducing this issue, I could try to do a smaller sample.

**Comment by felangel** [comment]
Hi @FredJul 👋 
I believe this should be resolved in the latest version of flow_builder. Can you please let me know if you're still seeing this issue in v0.0.4, thanks!

**Reply by FredJul** [reply]
Sorry but no, all versions above 0.0.1-dev.9 (including the 0.0.4) are buggy on android in my case (mainly with the back button/gesture, but not only). Not sure why exactly, maybe there is an issue in my code, but for now I'm sticking to 0.0.1-dev.9.
